### PR TITLE
refac: Bug in assertion fix

### PR DESCRIPTION
**Pull request by johevemi**
# Description

https://app.zenhub.com/workspaces/mandalorian-5fd22a1a59e4740018f5ab5a/issues/gh/energinet-datahub/team-mandalorian/294

https://app.zenhub.com/workspaces/mandalorian-5fd22a1a59e4740018f5ab5a/issues/gh/energinet-datahub/team-mandalorian/307

https://app.zenhub.com/workspaces/mandalorian-5fd22a1a59e4740018f5ab5a/issues/gh/energinet-datahub/team-mandalorian/303

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

**Review by JesperGomes — APPROVED** [review]

File: source/databricks/calculation_engine/tests/features/utils/assertion.py [hunk 1]
```diff
@@ -39,6 +39,10 @@ def assert_output(
     if "result_id" in expected_result.columns:
         columns_to_skip.append("result_id")
 
+    # Sort actual_result and expected_result
+    actual_result = actual_result.sort(actual_result.columns)
+    expected_result = expected_result.sort(expected_result.columns)
+
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/tests/features/utils/assertion.py:45]: What are they ordered by when using sort?

File: source/databricks/calculation_engine/tests/helpers/data_frame_utils.py [hunk 2]
```diff
@@ -47,6 +47,17 @@ def assert_dataframes_equal(actual: DataFrame, expected: DataFrame) -> None:
     ), "Dataframes data are not equal"
 
 
+def assert_no_duplicates(df: DataFrame) -> None:
+    original_count = df.count()
+    distinct_count = df.dropDuplicates().count()
+    assert original_count == distinct_count, "The DataFrame contains duplicate rows"
+
+
+def show_duplicates(df: DataFrame) -> DataFrame:
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/tests/helpers/data_frame_utils.py:56]: Why are there duplicates in the first place?

Comment (johevemi) [reply source/databricks/calculation_engine/tests/helpers/data_frame_utils.py:56]: in expected it is a personal error by the one creating the expected data.
and actual could be a code error

File: source/databricks/calculation_engine/tests/features/utils/assertion.py (hunk 1 above)
Comment (johevemi) [reply source/databricks/calculation_engine/tests/features/utils/assertion.py:45]: the column's order orders them

**Pull request merged**